cyhy_report/stakeholders/stakeholders.py

```diff
@@ -105,6 +119,13 @@ def generate_stakeholders_csv(db):
                 if org["_id"] in CI_sectors[sector]:
                     org_CI_sector = sector
                     break
+        
+        org_state = org["agency"]["location"].get("state", "N/A")
+        # Check the state against the REGION_MAPPING
+        for region_name, states in REGION_MAPPING.items():
+            if org_state in states:
+                org_region = region_name
+                break
```

Comment:
Rather than having to execute a `for` loop for every stakeholder, I'd modify your dictionary to something like this:
```python
REGION_MAPPING = {
    "AL": 4,
    "AK": 10,
     .
     .
     .
    "WY": 8,
}
```
I have it ordered alphabetically by state in my example snippet above, but ordered by region number and then state would also be fine by me.

With a mapping like that, you could do this to get the region:
```python
org_region = REGION_MAPPING.get(org_state, "N/A")
if org_region != "N/A":
    org_region = "Region {}".format(org_region)
```

You could get rid of the `if` statement there if you don't mind seeing `Region N/A` in your output:
```python
org_region = "Region {}".format(REGION_MAPPING.get(org_state, "N/A"))
```

Alternatively, you could just leave out the "Region X" string altogether and only output the region number or "N/A":
```python
org_region = REGION_MAPPING.get(org_state, "N/A")
```

Reply:
Resolved in https://github.com/cisagov/cyhy-reports/pull/115/commits/d2bec406fb7a9483404d69c4bef1146e7c2150d1 (see also https://github.com/cisagov/cyhy-reports/pull/115#issuecomment-1981574724).